Soccar [Client]/src/nl/soccar/ui/input/Keyboard.java

```diff
@@ -22,16 +27,32 @@ private Keyboard() {
         pressedKeys = new ArrayList<>();
     }
 
+    /**
+     * Method that checks if the given key is pressed.
+     * 
+     * @param code The keycode that needs to be checked.
+     * @return 

```

Comment:
* Method that checks if the given key **is already present in pressedKeys list**.

@return **boolean True if already pressed, false if not in pressedKeys list.**